test/test_producer.py

```diff
@@ -81,16 +86,22 @@ def test_kafka_producer_gc_cleanup():
 
 
 @pytest.mark.skipif(not env_kafka_version(), reason="No KAFKA_VERSION set")
-@pytest.mark.parametrize("compression", [None, 'gzip', 'snappy', 'lz4'])
+@pytest.mark.parametrize("compression", [None, 'gzip', 'snappy', 'lz4', 'zstd'])
 def test_kafka_producer_proper_record_metadata(kafka_broker, compression):
+    if compression == 'zstd' and env_kafka_version() < (2, 1, 0):
+        return
     connect_str = ':'.join([kafka_broker.host, str(kafka_broker.port)])
+    env_version = env_kafka_version()
+    api_version = env_version if env_version >= (2, 1, 0) else None
     producer = KafkaProducer(bootstrap_servers=connect_str,
                              retries=5,
+                             api_version=api_version,
                              max_block_ms=30000,
                              compression_type=compression)
     magic = producer._max_usable_produce_magic()
 
     # record headers are supported in 0.11.0
+
```

Comment:
Is this newline intended or accidental inclusion?

Reply:
Removed in relevant PR